### PR TITLE
Feature/mantl repackaging wiki

### PR DESCRIPTION
**Pull request by TanyaCouture**
Planning document for updating ansible roles to rpm packages.

### CODE REVIEW

**Comment by sehqlr** [comment]
@TanyaCouture and I started work on this. The file could use some reformatting, but the basic outline is there

File: Packaging-Roadmap-for-Mantl.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    - httpd-tools
+    - nc
+    - openssh
+    - policycoreutils-python
+    - epel-release
+    - unzip
+  - Ansible defaults: selinux with permissive policy and central configuration
+  - Ansible handlers: update-ca-trust -> run `update-ca-trust` cmd in package script when needed
+  - Ansible main tasks
+    - set timezone to UTC -> `ln -sf /etc/localtime /usr/share/zoneinfo/Etc/UTC`
```
Comment (stevendborrelli) [inline Packaging-Roadmap-for-Mantl.md]: Timezone can be done in cloud_init at boot.

File: Packaging-Roadmap-for-Mantl.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    - create `/etc/mantl` to hold metadata for state of cluster pre-consul boot
+    - j2 template for `/etc/hosts` -> consul template for `/etc/hosts`
```
Comment (stevendborrelli) [inline Packaging-Roadmap-for-Mantl.md]: Maybe we can skip the host population and add .node.consul to the dns search path.

File: Packaging-Roadmap-for-Mantl.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    - make tls directory -> `mkdir -p /etc/nginx/ssl && chmod 0700 /etc/nginx/ssl`
+    - deploy tls files
+    - encrypt admin password -> look at mantl PR#1058 for adding passwd to consul
+- consul: **mantl-consul**
+  - Question: could this be in **mantl-common**?
+- dnsmasq: **mantl-dnsmasq**
+  - PR#26
+
+*Roles for controls*
+- vault: **mantl-vault**
```
Comment (stevendborrelli) [inline Packaging-Roadmap-for-Mantl.md]: Vault only runs on a handful of machines

Comment (sehqlr) [inline Packaging-Roadmap-for-Mantl.md]: So, it needs to be on control nodes only. Are there any other parts of the base config that are control nodes only? Then maybe we could roll that into a **mantl-control** package

File: Packaging-Roadmap-for-Mantl.md (hunk 1 above)
Comment (sehqlr) [inline Packaging-Roadmap-for-Mantl.md]: I'll look into the cloud_init features we can leverage

**Comment by sehqlr** [comment]
I realized that this should be on the wiki, so this PR isn't needed. d'oh!

**Pull request closed without merging**